tests/test_window_utils.py

```diff
@@ -8,6 +8,7 @@
 class TestNewWindow(DeferrableTestCase):
 
     def tearDown(self):
+        yield 100
```

Comment:
Could you add a comment why the wait is necessary here and reference the upstream issue? We won't be able to remove this for a while until we drop compatibility with affected builds.

Reply:
Done.